benchexec/resources.py

```diff
@@ -157,64 +171,64 @@ def _get_cpu_cores_per_run0(coreLimit, num_of_threads, use_hyperthreading, allCp
             sys.exit("Core assignment is unsupported because siblings {0} of core {1} are not usable. Please always make all virtual cores of a physical core available.".format(siblings_set.difference(all_cpus_set), core))
 
     # Second, compute some values we will need.
-    package_count = len(cores_of_package)
-    packages = sorted(cores_of_package.keys())
+    unit_count = len(cores_of_unit)
+    units = sorted(cores_of_unit.keys())
     coreLimit_rounded_up = int(math.ceil(coreLimit / core_size) * core_size)
     assert coreLimit <= coreLimit_rounded_up < (coreLimit + core_size)
 
-    packages_per_run = int(math.ceil(coreLimit_rounded_up / package_size))
-    if packages_per_run > 1 and packages_per_run * num_of_threads > package_count:
-        sys.exit("Cannot split runs over multiple CPUs and at the same time assign multiple runs to the same CPU. Please reduce the number of threads to {0}.".format(package_count // packages_per_run))
+    units_per_run = int(math.ceil(coreLimit_rounded_up / unit_size))
+    if units_per_run > 1 and units_per_run * num_of_threads > unit_count:
+        sys.exit("Cannot split runs over multiple CPUs and at the same time assign multiple runs to the same CPU. Please reduce the number of threads to {0}.".format(unit_count // units_per_run))
 
-    runs_per_package = int(math.ceil(num_of_threads / package_count))
-    assert packages_per_run == 1 or runs_per_package == 1
-    if packages_per_run == 1 and runs_per_package * coreLimit > package_size:
-        sys.exit("Cannot run {} benchmarks with {} cores on {} CPUs with {} cores, because runs would need to be split across multiple CPUs. Please reduce the number of threads.".format(num_of_threads, coreLimit, package_count, package_size))
+    runs_per_unit = int(math.ceil(num_of_threads / unit_count))
+    assert units_per_run == 1 or runs_per_unit == 1
+    if units_per_run == 1 and runs_per_unit * coreLimit > unit_size:
+        sys.exit("Cannot run {} benchmarks with {} cores on {} CPUs with {} cores, because runs would need to be split across multiple CPUs. Please reduce the number of threads.".format(num_of_threads, coreLimit, unit_count, unit_size))
```

Comment:
For each occurrence of `CPUs` in this message, please write `CPUs/memory regions` instead. Otherwise it will be confusing for users.

Reply:
I have replaced the same, and also replaced the term `unit` to `CPU/memory regions` in other debug statements for better understanding.